src/main/java/org/folio/service/export/storage/AWSStorageServiceImpl.java

```diff
@@ -0,0 +1,57 @@
+package org.folio.service.export.storage;
+
+import com.amazonaws.HttpMethod;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.amazonaws.services.s3.model.GeneratePresignedUrlRequest;
+import java.net.URL;
+import java.util.Date;
+import org.springframework.stereotype.Service;
+
+/**
+ * Retrieve files that are stored in AWS S3
+ */
+@Service
+public class AWSStorageServiceImpl implements ExportStorageService {
+
+  AmazonS3 s3Client;
```

Comment:
Idea for refactoring - maybe we don't need to keep s3Client as a filed and get it each time with builder, in this case service will be stateless and it can be used as a spring singleton bean?

Reply:
We will wrap the AmazonS3ClientBuilder into custom class for testing/mocking needs